arch/risc-v/include/qemu-rv/irq.h

```diff
@@ -33,30 +33,7 @@
 
 /* Map RISC-V exception code to NuttX IRQ */
 
-/* IRQ 0-15 : (exception:interrupt=0) */
-
-#define QEMU_RV_IRQ_IAMISALIGNED  (0) /* Instruction Address Misaligned */
```

Comment:
should we remove other chip definition too

Reply:
Yes, I'll submit it later